src/rleCompress.js

```diff
@@ -19,6 +19,28 @@
  */
 function rleCompress(source) {
   // write code here
+  let compressedString = '';
+  let i = 0;
+  let amountFollowingLetters = 0;
+
+  while (i < source.length) {
```

Comment:
i is used only in this loop. Why not a simple for loop instead of this while?

Reply:
Done

Reply:
beacouse I like loop while(....

Reply:
Roman, all checks have failed! I'll find mistake...